.github/workflows/test.yaml

```diff
@@ -0,0 +1,19 @@
+name: Test Components
+
+on:
+    pull_request_target:
+        types: [opened, reopened]
+
+jobs:
+    run-tests:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+              with:
+                  persist-credentials: false # <--- this
```

Comment:
What does that comment mean 🤔 ?

Reply:
tbh I copied it from all other actions. I will remove it for all of them :D